src/cmd-ore-wrapper

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# NOTE: PYTHONUNBUFFERED is set in the entrypoint for unbuffered output
+#
+# This is a generic wrapper around 'ore' commands and serves
+# as a replacement for 'cmd-*-replicate' and 'cmd-buildextend-*'.
+
+import logging as log
+import sys
+
+from cosalib.cli import (
+    cloud_clis,
+    get_cloud_cli,
+    get_cloud_ore_cmds,
+    BuildCli
+)
+from cosalib.qemuvariants import get_qemu_variant
+
+if __name__ == '__main__':
+    log.basicConfig(
+        format='[%(asctime)s  %(levelname)s]: %(message)s',
+        level=log.INFO)
+
+    parser = BuildCli(
+        description="""interface for running commands against ore
+Each target has its own sub options. To access them us:
+    '--target <target> --help'
+""",
+        add_help=False,
+    )
+
+    # Determine if this a symlink, default to that.
+    sys_target = None
+    replicate = False
+    build_build = False
```

Comment:
This made me chuckle :grin:

Reply:
`build_build` make wonder about renaming `cosalib.build` to `cosalib.artifact` Perhaps in another PR.